app/controllers/container_controller.rb

```diff
@@ -74,6 +74,13 @@ def show_timeline
 
   def explorer
     @display = params[:display] || "main" unless control_selected?
+    @record = Container.find_by_id(from_cid(params[:id]))
+    show_timeline if @display == "timeline"
+    if @display == "performance"
+      @showtype = "performance"
+      drop_breadcrumb(:name => "#{@record.name} Capacity & Utilization", :url => "/container/show/#{@record.id}?display=#{@display}&refresh=n")
+      perf_gen_init_options               # Initialize perf chart options, charts will be generated async
```

Comment:
you have angered rubocop, please fix spacing before comment on L82 and length of L81 before we are all done for it